packages/storybook/src/stories/ino-nav-menu/ino-nav-menu.stories.ts

```diff
@@ -0,0 +1,77 @@
+import { Components } from '@inovex.de/elements';
+import { Meta } from '@storybook/web-components';
+import { html } from 'lit-html';
+import { TemplateGenerator } from '../template-generator';
+import { decorateStoryWithClass } from '../utils';
+import './ino-nav-menu.scss';
+
+export default {
+  title: `Structure/ino-nav-menu`,
+  component: 'ino-nav-menu',
+  decorators: [(story) => decorateStoryWithClass(story, 'story-ino-nav-menu')],
+  args: {
+    sections: ['Section 1', 'Section 2', 'Section 3'],
+    activeSection: 'section-1',
+    menuTitle: 'Sections',
+  },
+} as Meta<Components.InoNavMenu>;
+
+const activeSectionChanged = (x: CustomEvent<string>) =>
+  console.log('active section changed: ' + x.detail);
```

Comment:
This message could be made visible by showing it in a snackbar. See [ino-dialog closing action](https://elements.inovex.de/version/latest/?path=/docs/structure-ino-dialog--playground)